### PR TITLE
rocky-linux/rocky-tools#41 Add note to README about OpenJDK packages

### PR DESCRIPTION
**Pull request by klaus-tux**
As suggested by @pajamian in https://github.com/rocky-linux/rocky-tools/issues/41#issuecomment-867273803 I added a note about issues with the OpenJDK packages to the README.
Feel free to merge or adapt.

### CODE REVIEW

File: migrate2rocky/README.md [hunk 1]
```diff
@@ -56,6 +56,16 @@ on reboot but the other ones will remain until they are removed or replaced by
...
+#### Symbolic links to Java programs in `/etc/alternatives` are gone
+
+After migrating a system with an OpenJDK package installed you might encounter
+that java does not work any more. Apparently there is a bug in the OpenJDK
+packages that leads to losing the symbolic links in `/etc/alternatives` during
+the migration. Details can be found
+[here](https://github.com/rocky-linux/rocky-tools/issues/41#issuecomment-867200639).
+There you can also find a simple script that recreates the package default
+alternatives.
+
```
Comment (pajamian) [inline migrate2rocky/README.md:68]: Link to the RedHat bugzilla:
https://bugzilla.redhat.com/show_bug.cgi?id=1976053

Add the one-liner fix directly here:
`rpm -qa --scripts java-{1.8.0,11}-openjdk-{headless,devel} | sed -n '/postinstall/, /exit/{ /postinstall/! { /exit/ ! p} }' | sh`

...then you don't need to reference the script on the issues page.

**Review by pajamian — CHANGES_REQUESTED** [review]

**Comment by klaus-tux** [comment]
Hi @pajamian ! Thank you for your deeper analysis of the problem and the bug report! I updated the pull request accordingly.

**Comment by pajamian** [comment]
Pulled, thanks :-)

**Comment by klaus-tux** [comment]
Great! Thank you :-)

**Pull request merged**